src/components/Book.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = props => {
+  const { id, title, category } = props;
+  return (
+    <tr>
+      <th>{id}</th>
+      <th>{title}</th>
+      <th>{category}</th>
+    </tr>
+  );
+};
+
```

Comment:
This component is expected to accept `Book` object as prop. Meaning you'll have to pass the whole object instead of the individual properties
[[ see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png) ]